pkg/deployments/tasks/20230215-single-recipient-gauge-factory-v2/readme.md

```diff
@@ -3,6 +3,8 @@
 Deployment of the `SingleRecipientGaugeFactory`, for liquidity gauges which forward their entire BAL allowance onto a single address.
 This version adds a weight cap to single recipient gauges, and supersedes [single recipient gauge factory V1](../deprecated/20220325-single-recipient-gauge-factory/).
 
+Additionally, this version also supports the recipient implementing the `depositToken` function (from the `FeeDistributor` interface), directly deposoting the tokens in such a distributor instead of requiring a manual step after minting.
```

Comment:
```suggestion
Additionally, this version also supports the recipient implementing the `depositToken` function (from the `FeeDistributor` interface), directly depositing the tokens in such a distributor instead of requiring a manual step after minting.
```